modules/headline.py

```diff
@@ -12,16 +14,18 @@ def __init__(self):
 
     def extract_headline(self):
         self.net_con=True #Expecting Internet Connection to be working initially
+        
         try:
```

Comment:
No need for a try except

Reply:
Removed, although try and except seems like a good practice to me.

Reply:
Hey there is a need for try except I believe. When internet is not connected it gives a hell of a long error messege. Also hampers other functions task

Reply:
Downloading and parsing the article cant go wrong when there is an internet connection. Testing for internet connection should be done in __init__. So here, the try-except is not necessary.